### PR TITLE
[DEVX-1889] Update restproxy creds

### PR DESCRIPTION
**Pull request by sdanduConf**
This switches rest proxy to use it's own LDAP user and adds the user to LDAP.  In the future, may need to have specific role bindings for rest proxy, so this enables that and is a better separation vs. using connect admin.

Testing: With this change, brought up cp-demo and walked through security section in documentation with all rest proxy commands.

Includes on other minor change to add a wildcard for monitor-interceptors in connect classpath (since there is only one jar, this just makes it easier in case version matching is not exact)

### CODE REVIEW

File: scripts/security/ldap_users/21_restAdmin.ldif [hunk 1]
```diff
@@ -0,0 +1,15 @@
+dn: cn=restAdmin,ou=users,{{ LDAP_BASE_DN }}
```
Comment (ybyzek) [inline scripts/security/ldap_users/21_restAdmin.ldif:1]: @sdanduConf for consistency, perhaps `restAdmin` user should also be added to the group ldif, similar to `connectAdmin` at https://github.com/confluentinc/cp-demo/blob/5.5.0-post/scripts/security/ldap_users/20_group_add.ldif#L26-L29

**Comment by ybyzek** [comment]
@sdanduConf not sure what happened.  I thought I had deleted the ksqldb-cli image and pulled the new one, but apparently I didn't.  Rerunning now.  (thanks @mikebin )

**Comment by ybyzek** [comment]
@sdanduConf I have validated the PR and implicitly 5.5.1 ;) Great work.

One additional change could be made in https://github.com/confluentinc/cp-demo/blob/5.5.0-post/scripts/validate/validate_bindings.sh to drive home the point that there are no RP role bindings

```
[21:55:10] ~/git/cp-demo(update_restproxy_creds) ✗: git diff
diff --git a/scripts/validate/validate_bindings.sh b/scripts/validate/validate_bindings.sh
index 9f386a6..556fc4f 100755
--- a/scripts/validate/validate_bindings.sh
+++ b/scripts/validate/validate_bindings.sh
@@ -25,6 +25,7 @@ SR_PRINCIPAL="User:schemaregistryUser"
 KSQLDB_ADMIN="User:ksqlDBAdmin"
 KSQLDB_USER="User:ksqlDBUser"
 C3_ADMIN="User:controlcenterAdmin"
+REST_ADMIN="User:restAdmin"
 CLIENT_PRINCIPAL="User:appSA"
 BADAPP="User:badapp"
 LISTEN_PRINCIPAL="User:clientListen"
@@ -33,7 +34,7 @@ docker-compose exec tools bash -c ". /tmp/helper/functions.sh ; mds_login $MDS_U
 
 ################################## Run through permutations #############################
 
-for p in $SUPER_USER_PRINCIPAL $CONNECT_ADMIN $CONNECTOR_SUBMITTER $CONNECTOR_PRINCIPAL $SR_PRINCIPAL $KSQLDB_ADMIN $KSQLDB_USER $C3_ADMIN $CLIENT_PRINCIPAL $BADAPP $LISTEN_PRINCIPAL; do
+for p in $SUPER_USER_PRINCIPAL $CONNECT_ADMIN $CONNECTOR_SUBMITTER $CONNECTOR_PRINCIPAL $SR_PRINCIPAL $KSQLDB_ADMIN $KSQLDB_USER $C3_ADMIN $REST_ADMIN $CLIENT_PRINCIPAL $BADAPP $LISTEN_PRINCIPAL; do
   for c in " " " --schema-registry-cluster-id $SR" " --connect-cluster-id $CONNECT" " --ksql-cluster-id $KSQLDB"; do
     echo
     echo "Showing bindings for principal $p and --kafka-cluster-id $KAFKA_CLUSTER_ID $c"
[21:55:14] ~/git/cp-demo(update_restproxy_creds) ✗: 
```

File: scripts/security/ldap_users/21_restAdmin.ldif (hunk 1 above)
Comment (sdanduConf) [reply scripts/security/ldap_users/21_restAdmin.ldif:1]: Added, thanks!

**Comment by sdanduConf** [comment]
thanks for the feedback/changes @ybyzek .  PR is updated for review.

**Comment by sdanduConf** [comment]
Just to note here: output from validate_bindings for rest_admin:

> Showing bindings for principal User:restAdmin and --kafka-cluster-id -_LNOLf6RTCfCvdnU6d2Bg  
>   Principal | Role | ResourceType | Name | PatternType  
> +-----------+------+--------------+------+-------------+
> 
> 
> Showing bindings for principal User:restAdmin and --kafka-cluster-id -_LNOLf6RTCfCvdnU6d2Bg  --schema-registry-cluster-id schema-registry
>   Principal | Role | ResourceType | Name | PatternType  
> +-----------+------+--------------+------+-------------+
> 
> 
> Showing bindings for principal User:restAdmin and --kafka-cluster-id -_LNOLf6RTCfCvdnU6d2Bg  --connect-cluster-id connect-cluster
>   Principal | Role | ResourceType | Name | PatternType  
> +-----------+------+--------------+------+-------------+
> 
> 
> Showing bindings for principal User:restAdmin and --kafka-cluster-id -_LNOLf6RTCfCvdnU6d2Bg  --ksql-cluster-id ksql-cluster
>   Principal | Role | ResourceType | Name | PatternType  
> +-----------+------+--------------+------+-------------+
> 
>

**Pull request merged**